components/clp-package-utils/clp_package_utils/general.py

```diff
@@ -278,6 +291,7 @@ def generate_container_config(
             clp_config.aws_config_directory,
             container_clp_config.aws_config_directory,
         )
+
```

Comment:
I am fine with a new line, but in case if it's not intended.

Reply:
let me revert since it's not strictly related to the PR